compiler/rustc_codegen_llvm/src/llvm_util.rs

```diff
@@ -282,6 +282,15 @@ pub(crate) fn to_llvm_features<'a>(sess: &Session, s: &'a str) -> Option<LLVMFea
         }
         // Filter out features that are not supported by the current LLVM version
         ("riscv32" | "riscv64", "zacas") if get_version().0 < 20 => None,
+        ("message-security-assist-extension12"
```

Comment:
```suggestion
        ("s390x", "message-security-assist-extension12"
```
this is tuple and the first field is target_arch.